src/pages/dar_collection_review/DarVoteReview.js

```diff
@@ -0,0 +1,6 @@
+import DarCollectionReview from "./DarCollectionReview";
+
+export default function DarVoteReview(props) {
```

Comment:
If the review component is only adding `readOnly` to props for DarCollectionReview then a better option is to just add `readOnly` to props on the routes page. I'll leave a comment there to show what I mean